postinstall/frontend_hangingio.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import json
+import re
+import argparse
+import subprocess
+import sys
+import logging
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Filter HangingIos by age.')
```

Comment:
A nit -- but can the description be changed? The utility is doing more than filtering -- it is actively dropping the cache.

Reply:
sure I will need to remove the pull request and update that info. Good Tip.